src/main/java/org/sagebionetworks/bridge/sdk/AdminClient.java

```diff
@@ -18,37 +18,44 @@
      *            Credentials used to sign up the user into Bridge.
      * @return true if success, false if failure.
      */
-    public boolean createUser(SignUpByAdmin signUp);
+    boolean createUser(SignUpByAdmin signUp);
```

Comment:
He's not reducing the visibility. Members in interfaces are always implicitly public.